src/Components/SearchModal/SearchPostList/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { useNavigate } from 'react-router-dom';
+import { StyledWrapper, StyledContainer, StyledNoContent } from './style';
+import Props from './type';
+
+const SearchPostList = ({ data }: Props) => {
+  const navigator = useNavigate();
+
+  const handlePostClick = (postId: string) => {
+    navigator(`/modal-detail/${postId}`);
+  };
+  return (
+    <StyledWrapper>
+      {data ? (
+        data.map((post) => (
+          <StyledContainer
+            key={post._id}
+            onClick={() => handlePostClick(post._id)}
+          >
+            <h1>{post.title}</h1>
+            <p> {post.createdAt.slice(0, 10)}</p>
```

Comment:
이거 UTC 기준이라, 9시간 시차 때문에 날짜가 하루 차이 날 때가 가끔 있을거에요! 제가 Utils/UTCtoKST에 `convertUtcToKstDate` 함수 구현해놨으니까 이거 사용해주시면 좀 더 정확하게 날짜 나올거에요 ㅎㅎ

```suggestion
            <p> {convertUtcToKstDate(post.createdAt)}</p>
```